core-bundle/src/Resources/contao/library/Contao/Model.php

```diff
@@ -409,12 +403,6 @@ public function current()
 	 */
 	public function save()
 	{
-		// Deprecated call
-		if (\func_num_args() > 0)
-		{
-			throw new \InvalidArgumentException('The $blnForceInsert argument has been removed (see system/docs/UPGRADE.md)');
```

Comment:
Why remove this? It still fulfills its purpose even in Contao 5 IMHO.

Reply:
Fine for me as well, though I'd prefer cleaning this up now.

Reply:
Yes, we probably should. 👍